src/frontend/app/shared/components/list/list-types/services-wall/service-instance-card/service-instance-card.component.html

```diff
@@ -24,7 +24,18 @@
     <app-meta-card-key>Last Updated</app-meta-card-key>
     <app-meta-card-value>{{ serviceInstanceEntity?.entity.last_operation?.updated_at | date:'medium' }} </app-meta-card-value>
   </app-meta-card-item>
-
+  <app-meta-card-item>
+    <app-meta-card-key>Dashboard URL</app-meta-card-key>
+    <app-meta-card-value>
+      <div *ngIf="hasDashboardUrl(); then dashboardUrl; else none"></div>
+      <ng-template #dashboardUrl>
+        <a [href]="getDashboardUrl()" target="_blank" appClickStopPropagation>
+          <mat-icon>launch</mat-icon>
+        </a>
+      </ng-template>
+      <ng-template #none>{{ getDashboardUrl() }}</ng-template>
```

Comment:
Just add 'None' into the template here, no need to get angular to do the work for us.